xml/System/String.xml

```diff
@@ -4793,15 +4793,17 @@ In .NET Core, sorting and comparison operations are based on [Version 8.0.0 of t
 ## How arguments are formatted  
  Format items are processed sequentially from the beginning of the string. Each format item has an index that corresponds to an object in the method's argument list. The <xref:System.String.Format%2A> method retrieves the argument and derives its string representation as follows:  
   
--   If the argument is `null`, the method inserts <xref:System.String.Empty?displayProperty=nameWithType> into the result string. You don't have to be converned with handling an <xref:System.NullReferenceException> for null arguments. 
+-   If the argument is `null`, the method inserts <xref:System.String.Empty?displayProperty=nameWithType> into the result string. You don't have to be concerned with handling an <xref:System.NullReferenceException> for null arguments. 
   
--   If you call the <xref:System.String.Format%28System.IFormatProvider%2CSystem.String%2CSystem.Object%5B%5D%29> overload and the `provider` parameter implements the <xref:System.ICustomFormatter> interface, the argument is passed to the `provider` object's <xref:System.ICustomFormatter.Format%28System.String%2CSystem.Object%2CSystem.IFormatProvider%29?displayProperty=nameWithType> method. If the format item includes a *formatString* argument, it is passed as the first argument to the method. If the <xref:System.ICustomFormatter> implementation is able to provide formatting services, it returns the string representation of the argument; otherwise, it returns `null` and the next step executes.  
+-   If you call the <xref:System.String.Format%28System.IFormatProvider%2CSystem.String%2CSystem.Object%5B%5D%29> overload and the `provider` object's <xref:System.IFormatProvider.GetFormat%2A?displayProperty=nameWithType> implementation returns non-null <xref:System.ICustomFormatter> implementation, the argument is passed to its <xref:System.ICustomFormatter.Format%28System.String%2CSystem.Object%2CSystem.IFormatProvider%29?displayProperty=nameWithType> method. If the format item includes a *formatString* argument, it is passed as the first argument to the method. If the <xref:System.ICustomFormatter> implementation is available and produces non-null string, that string is returned as the string representation of the argument; otherwise, the next step executes.  
   
 -   If the argument implements the <xref:System.IFormattable> interface, its <xref:System.IFormattable.ToString%2A?displayProperty=nameWithType> implementation is called.  
   
--   The argument's parameterless `ToString` method, which is either overridden or inherited from the <xref:System.Object> class, is called.  
+-   The argument's parameterless `ToString` method, which is either overridden or inherited from a base class, is called.  
```

Comment:
again a nit unrelated to your PR: is overridden is wrong; it should be "which either overrides or inherits from a base class implementation".